ios/ParentBuddyApp.xcodeproj/project.pbxproj

```diff
@@ -8,19 +8,21 @@
 
```

Comment:
Hope you made all these changes intentionally otherwise I would suggest to double check.

Reply:
@Akhror yes it's change due configuration